.github/workflows/upload-driver-packages.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           files: "drivers"
           json: true
       - id: set-outputs
-        run: echo "matrix=${{ steps.changed-drivers.outputs.all_modified_files }}" >> "$GITHUB_OUTPUT"
+        run: echo "matrix=${{ steps.changed-drivers.outputs.all_modified_files }}" >> $GITHUB_OUTPUT
```

Comment:
Why are we no longer quoting the output here?

Reply:
shouldn't have been there in the first place

Reply:
just decided to make it consistent with elsewhere